projects/demo/src/modules/markup/typography/typography.template.html

```diff
@@ -19,6 +20,11 @@ <h3 class="tui-text_body-l tui-space_top-0 tui-space_bottom-4">
                         tui-text_h1
                     </tui-doc-copy>
                 </td>
+                <td class="td">
+                    <tui-doc-copy cdkCopyToClipboard="--tui-font-heading-1">
```

Comment:
nit:
```suggestion
                    <tui-doc-copy cdkCopyToClipboard="var(--tui-font-heading-1)">
```

Reply:
:thinking: Hmm, I think we should copy to clipboard exactly what we see. Maybe wrap with var() both lines ?

Reply:
We have the same logic with colors https://taiga-ui.dev/colors 

If you click any color name, it will be copied with `var(...` prefix

Reply:
ok, I did the same logic)